src/di.php

```diff
@@ -306,6 +307,14 @@
     $config = $di['config'];
     $options = $config['twig'];
 
+    // Get internationalisation settings from config, or use sensible defaults for
+    // missing required settings.
+    $locale = $config['i18n']['locale'] ?? 'en_US';
+    $timezone = $config['i18n']['timezone'] ?? 'UTC';
+    $date_format = $config['i18n']['date_format'] ? strtoupper($config['i18n']['date_format']) : 'MEDIUM';
```

Comment:
This won't work, you're trying to reference an undefined var. Will have to wrap them in an isset

``` PHP
$date_format =  isset($config['i18n']['date_format']) ? strtoupper($config['i18n']['date_format']) : 'MEDIUM';
$time_format =  isset($config['i18n']['time_format']) ? strtoupper($config['i18n']['time_format']) : 'SHORT';
```

Reply:
I've added an `!empty()` check on those variables as discussed. 🙂